### PR TITLE
PCPARTS-14 | Connect front to graphql

### PR DESCRIPTION
**Pull request by monika-klokosinska-intive**
## PR Checklist

-   [x] Title format: **PCPARTS-XXX | Title Case Description**
-   [x] **README** is up to date OR not necessary

## JIRA Ticket

[JIRA-14](https://exdorf.atlassian.net/browse/PCPARTS-14)

## Description
Scope:
 - connect to local graphQL
 - get all available cpus and motherboards
 - display motherboards and cpus using a generic select component

### CODE REVIEW

File: frontend/type/Motherboard.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default interface Motherboard {
+    _id: string,
+    ram_slots: 4,
```
Comment (IntiveMichal) [inline frontend/type/Motherboard.ts]: why just 4?

File: frontend/src/components/SelectorView/index.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+interface componentProps {
+    motherboards: Motherboard[]
+    cpus: CPU[]
+}
+
+function SelectorView({ motherboards, cpus }: componentProps) {
+
+    return (
+        <div>
```
Comment (IntiveMichal) [inline frontend/src/components/SelectorView/index.tsx]: If its not styled, can be replaced with React.Fragment or with <>

Comment (monika-klokosinska-intive) [reply frontend/src/components/SelectorView/index.tsx]: @IntiveMichal I added some basic grid, will expand on it in the coming days!

File: frontend/type/Motherboard.ts (hunk 1 above)
Comment (monika-klokosinska-intive) [reply frontend/type/Motherboard.ts]: @IntiveMichal no idea... I replaced the `4` with `number`.

**Review by IntiveMichal — APPROVED** [review]

**Pull request merged**